### PR TITLE
Accordion: Replace zustand state management with useReducer

### PR DESCRIPTION
**Pull request by ItsJonQ**
This update replaces the state management of the `Accordion` component with React's `useReducer`.
This can be used in the Accordion story of Storybook.

Clicking on items should collapse other items in the same group.
The two Accordions should synchronize.

Screen Capture on 2021-01-12 at 13-34-59

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -25495,11 +25495,6 @@ yurnalist@^1.1.2:
     strip-ansi "^5.2.0"
     strip-bom "^4.0.0"
 
-zustand@^3.3.1:
-  version "3.3.1"
-  resolved "https://registry.yarnpkg.com/zustand/-/zustand-3.3.1.tgz#de5c4b51112b84e0f819d8b3f336fbfbc087d758"
-  integrity sha512-o0rgrBsi29nCkPHdhtkAHisCIlmRUoXOV+1AmDMeCgkGG0i5edFSpGU0KiZYBvFmBYycnck4Z07JsLYDjSET9g==
-
```
Comment (saramarcondes) [inline yarn.lock:25502]: Is this correct?

**Review by saramarcondes — APPROVED** [review]
@ItsJonQ LGTM but let me know if you'd like to pair on adding types to this 👍🏼

File: yarn.lock (hunk 1 above)
Comment (ItsJonQ) [reply yarn.lock:25502]: I think so? I think it did it automatically after I ran `yarn start`. I can double check

**Comment by ItsJonQ** [comment]
> like to pair on adding types to this
I'd love to 🍐 on types :D

File: yarn.lock (hunk 1 above)
Comment (ItsJonQ) [reply yarn.lock]: I just merged in latest `master`

**Pull request merged**